pkg/plugins/golang/v2/scaffolds/internal/templates/makefile.go

```diff
@@ -60,6 +60,11 @@ IMG ?= {{ .Image }}
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true"
 
+# run container engine commands (such as building and pushing)
+OCI_ENGINE ?= docker
+# run commands against a kubernetes cluster
+KUBECTL ?= kubectl
+
```

Comment:
You are here to change the go/v2 plugin. 
This kind of change should be addressed to go/v3 which is the latest version.